pyaedt/generic/grpc_plugin_dll.py

```diff
@@ -59,7 +59,19 @@ def SetPyObjCalbacks(CreateAedtObj, CreateAedtBlockObj, GetAedtObjId):
     AedtAPI.SetPyObjCalbacks(callbackToCreateObj, callbackCreateBlock, callbackGetObjID)
 
 
-AedtAPI.CreateAedtApplication.argtypes = c_wchar_p, c_int, c_bool, c_bool
+# Find the version of AEDT from product info file
```

Comment:
@maxcapodi78 I'm not sure if there is a better way to get the version information here in the plugin initialization, but we need some way to handle the difference in argument types between 23R2 and 24R1.

Reply:
@Samuelopez-ansys - can you review and merge this PR if it looks good to you? It's holding up some related development.

Reply:
@jsalant22 Maybe you can take a look at this PR? Thanks!